### PR TITLE
generic error value is not really optional, so don't declare it optional

### PR DESCRIPTION
**Pull request by aliak00**

### CODE REVIEW

File: SchibstedAccount/Core/GenericError.swift [hunk 1]
```diff
@@ -20,16 +20,13 @@ extension GenericError {
 
 extension GenericError.Unexpected: CustomStringConvertible {
     var description: String {
-        if let string = self.string {
-            return "\(type(of: self))! \(string)"
-        }
-        return "\(type(of: self))!"
+        return "\(type(of: self))! \(string)"
```
Comment (lrossi) [inline SchibstedAccount/Core/GenericError.swift]: Even though it should never fail, can we avoid the bang here? Maybe something like `?? "n/a"`?

File: SchibstedAccount/Core/GenericError.swift [hunk 2]
```diff
@@ -38,9 +35,6 @@ extension GenericError {
 
 extension GenericError.WTF: CustomStringConvertible {
     var description: String {
-        if let string = self.string {
-            return "\(type(of: self))! \(string)"
-        }
-        return "\(type(of: self))!"
+        return "\(type(of: self))! \(string)"
```
Comment (lrossi) [inline SchibstedAccount/Core/GenericError.swift]: Same here

File: SchibstedAccount/Core/GenericError.swift (hunk 1 above)
Comment (aliak00) [reply SchibstedAccount/Core/GenericError.swift]: The bang is part of the output text :p 

i.e. "Error! something failed"

Comment (lrossi) [reply SchibstedAccount/Core/GenericError.swift]: Oh, true! 😆

**Pull request merged**